dub.sdl

```diff
@@ -29,9 +29,8 @@ subPackage {
     "src/ddmd/tokens.d" \
     "src/ddmd/utf.d"
 
-  // stringImportPaths cannot be used because it will make Dub extremely slow
-  // https://github.com/dlang/dub/issues/1199
-  dflags "-J$PACKAGE_DIR"
+  preGenerateCommands "\"$${DUB_PACKAGE_DIR}config.sh\" \"$${DUB_PACKAGE_DIR}generated/dub\" VERSION /etc"
+  stringImportPaths "generated/dub"
```

Comment:
Will this work when building a package that has DMD as a dependency? I had problems with this before, that's why I switch from `-J.` to `-J$PACKAGE_DIR`.